.github/workflows/ecosystem-ci.yml

```
@@ -107,53 +121,10 @@ jobs:
           client_payload: '{"ref":"${{ github.event.inputs.branch }}","repo":"web-infra-dev/rsbuild","suite":"-","suiteRefType":"precoded","suiteRef":"precoded"}'
 
       - id: get-ci-result
-        uses: actions/github-script@v7
+        uses: ./.github/actions/get-ci-result
```

`get-ci-result` is too general

```suggestion
        uses: ./.github/actions/eco-ci-result
```